### DISCUSSION TITLE
It's finally published to GitHub.

### DISCUSSION THREAD

**Discussion by Walidbenmoussa** [TomSchimansky/CustomTkinter#2345] [Showcase]
It's finally published to GitHub.

![image](https://github.com/TomSchimansky/CustomTkinter/assets/89534395/4b3aa4d7-e653-468e-8060-afce81aa6ea0)

You can now download CTk Theme Builder from my GitHub repo page. Please click [here](https://github.com/avalon60/ctk_theme_builder/releases/tag/v2.0.0). Download the  ctk_theme_builder-2.0.0.zip  file.

Installation instructions are available [here](https://github.com/avalon60/ctk_theme_builder/blob/develop/docs/installation.md).

Documentation is well under way and can be found [here](https://github.com/avalon60/ctk_theme_builder/blob/develop/docs/user_guide.md).

_Originally posted by @avalon60 in https://github.com/TomSchimansky/CustomTkinter/discussions/1659_

**Comment by dekrypted** [comment]
shut up and stop self advertising
